login/login.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <noscript>To play CardShifter, please either enable JavaScript or use a JavaScript-enabled browser.</noscript>
+        <form name = "login_information">
+            Server:
+            <input name = "server" type = "text" placeholder = "dwarftowers.com" />
```

Comment:
There's no need for a port field for now, just always use port 4243.

You might want to consider a `<select>` box with some pre-defined servers, then you could add both dwarftowers.com and 127.0.0.1 to that box.